aot/src/main/java/com/dylibso/chicory/aot/AotEmitters.java

```diff
@@ -104,7 +73,7 @@ public static void ELEM_DROP(AotContext ctx, AnnotatedInstruction ins, MethodVis
         asm.visitVarInsn(Opcodes.ALOAD, ctx.instanceSlot());
         asm.visitLdcInsn(index);
         asm.visitInsn(Opcodes.ACONST_NULL);
-        emitInvokeVirtual(asm, INSTANCE_SET_ELEMENT);
+        emitInvokeVirtual(asm, AotMethodsRefs.INSTANCE_SET_ELEMENT);
```

Comment:
Let's keep these static imported. It looks cleaner and makes the diff smaller.

Reply:
In all honesty, I personally prefer this notation, but I'm not fussy on it.